index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- `[OPTIONAL]` You did a great job by using polite words to ask for changes. Please it is important to mention why `title` matters a lot for a website, so that a student should know the genuine reason.